examples/terraform-gcp-example/main.tf

```diff
@@ -0,0 +1,47 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# CONFIGURE THE GOOGLE CLOUD PROVIDER
+# ---------------------------------------------------------------------------------------------------------------------
+
+provider "google" {
+  project = "${var.project}"
+  version = "~> 1.16"
+}
+
+# ---------------------------------------------------------------------------------------------------------------------
+# DEPLOY A CLOUD INSTANCE RUNNING UBUNTU
+# See test/terraform_gcp_example_test.go for how to write automated tests for this code.
+# ---------------------------------------------------------------------------------------------------------------------
+
+resource "google_compute_instance" "example" {
+  name         = "${var.instance_name}"
+  machine_type = "${var.machine_type}"
+  zone         = "${var.zone}"
+
+  boot_disk {
+    initialize_params {
+      image = "${data.google_compute_image.ubuntu.self_link}"
```

Comment:
Yes, its possible and I'll change. For some reason, I deliberately did this, because I thought the verbosity might be clearer for people reading the tests.